chart-testing.yaml

```
@@ -0,0 +1,115 @@
+package:
+  name: chart-testing
+  version: "3.12.0"
+  epoch: 0
+  description: Tool for testing Helm charts, used for linting and testing pull requests.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - git
+      - helm
+      - kubectl
+      - openssh-client
+      - py3-yamale-bin
+      - yamllint
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/helm/chart-testing
+      tag: v${{package.version}}
+      expected-commit: d6991035017d7ac0e3dec3d1b5ad2e5f18674b32
+
+  - uses: go/build
+    with:
+      modroot: .
+      packages: ./ct
+      output: ct
+      ldflags: "-X github.com/helm/chart-testing/v3/ct/cmd.Version=${{package.version}} -X github.com/helm/chart-testing/v3/ct/cmd.BuildDate=$(date -u +%Y-%m-%dT%H:%M:%SZ) -X github.com/helm/chart-testing/v3/ct/cmd.GitCommit=$(git rev-parse --short HEAD)"
```

we can move the ldflags inline to make it more readable and SOURCE_DATE_EPOCH is required to make the package reproducible. 
```suggestion
      ldflags: |
        -X github.com/helm/chart-testing/v3/ct/cmd.Version=${{package.version}} 
        -X github.com/helm/chart-testing/v3/ct/cmd.BuildDate=$(date -u -d "@$SOURCE_DATE_EPOCH" +"%Y-%m-%dT%H:%M:%SZ")
        -X github.com/helm/chart-testing/v3/ct/cmd.GitCommit=$(git rev-parse --short HEAD)"
```